content/authorization/what-do-you-get/accessgroups/accessgroups/transportOgLagring/_index.en.md

```
@@ -11,12 +11,12 @@ aliases:
 
 |**Tilgangspakker i kategorien**|
 |---|
-|[Veitransport](https://docs.altinn.studio/authorization/what-do-you-get/accessgroups/accessgroups/transportoglagring/#veitransport)|
-|[Transport i rør ](https://docs.altinn.studio/authorization/what-do-you-get/accessgroups/accessgroups/transportoglagring/#transport-i-rør)|
-|[Sjøfart](https://docs.altinn.studio/authorization/what-do-you-get/accessgroups/accessgroups/transportoglagring/#sjøfart)|
-|[Lufttransport](https://docs.altinn.studio/authorization/what-do-you-get/accessgroups/accessgroups/transportoglagring/#lufttransport)|
-|[Jernbanetransport](https://docs.altinn.studio/authorization/what-do-you-get/accessgroups/accessgroups/transportoglagring/#lufttransport)|
-|[Lagring og andre tjenester tilknyttet transport](https://docs.altinn.studio/authorization/what-do-you-get/accessgroups/accessgroups/transportoglagring/#lagring-og-andre-tjenester-tilknyttet-transport)|  
+|[Veitransport](#veitransport)|
+|[Transport i rør ](#transport-i-rør)|
```

markdownlint-cli2 (0.18.1)

Spaces inside link text

(MD039, no-space-in-links)